e2etest/zt_amanaged_disks_test.go

```diff
@@ -118,7 +119,7 @@ func TestManagedDisks_SnapshotOAuth(t *testing.T) {
 }
 
 // Service issue causes occasional flickers in feature functionality; enough that testing is problematic. Temporarily disabled until issue is resolved.
-func TestManagedDisks_OAuthRequired(t *testing.T) {
+func TestAAA(t *testing.T) {
```

Comment:
Why is test case name changed to "AAA" ?

Reply:
Rolled back the change.